3p/integration.js

```diff
@@ -337,7 +335,10 @@ function triggerResizeRequest(width, height) {
 }
 
 function triggerRenderStart() {
-  nonSensitiveDataPostMessage('render-start');
+  if (!window.context.isRenderStartSent) {
```

Comment:
@lannka what do you think on this? I think @jridgewell has a good point, and it's ok to postMessage twice. 
But on the other side, we feel that as we are able to prevent that from happening by one check, we probably should do that.